lib/classes/marked_date.dart

```diff
@@ -32,6 +32,10 @@ class MarkedDate implements MarkedDateInterface {
 
   @override
   TextStyle? getTextStyle() => this.textStyle;
+
+  @override
+  // TODO: implement hashCode
+  int get hashCode => super.hashCode;
```

Comment:
Could you tell the downside if we don't implement this?

Reply:
If you want to compare object to object, it's helpful.

It is auto generated by dart fix --apply.